src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -389,6 +508,35 @@ private static void CompareDateTimeResults(string actual, string expected)
             }
         }
 
+        private static void ValidateArrayResults(JsonDocument actualJsonDoc, JsonDocument expectedJsonDoc, string fieldName)
+        {
+            JsonElement.ArrayEnumerator actualEnumerater = actualJsonDoc.RootElement.EnumerateArray();
+            foreach (JsonElement expectedElement in expectedJsonDoc.RootElement.EnumerateArray())
+            {
+                actualEnumerater.MoveNext();
+                JsonElement actualElement = actualEnumerater.Current;
+                JsonElement actualValue;
+                JsonElement expectedValue;
+                actualElement.TryGetProperty(fieldName, out actualValue);
+                expectedElement.TryGetProperty(fieldName, out expectedValue);
+
+                if (fieldName.StartsWith(DATETIME_TYPE.ToLower()))
+                {
+                    DateTime actualDateTime = DateTime.Parse(actualValue.ToString(), CultureInfo.InvariantCulture, DateTimeStyles.None);
```

Comment:
I validates that the date returned from our GQL query matches what we expect based on the data base query we pre generate. The reason that we do parsing this way, from a string rather than directly from the object is because the format the `MySql` returns will not correctly parse using the built in `GetDate` functions. I can add a comment but this is the same validation we do for all the other types, just a trickier bit of parsing.